es-app/src/views/gamelist/ISimpleGameListView.cpp

```diff
@@ -118,15 +118,15 @@ bool ISimpleGameListView::input(InputConfig* config, Input input)
 			}
 
 			return true;
-		}else if(config->isMappedTo("right", input))
+		}else if(config->isMappedTo("right", input) || config->isMappedTo("rightshoulder", input))
```

Comment:
is this related to this commit ?

Reply:
Changing system while in game view is usually done with right and left arrows. In the grid view we can't do that since left and right arrows control the selector (along with up and down) so switching system is done with left and right shoulder. For not breaking user habits I decided to allow switch system with left and right shoulder for all the game view. If later we decide to allow the user to choose different game view style depending on the system it will also be really handly (for example an automatic option which display the grid view for systems with metadata and the basic view for systems without metadata, like the automatic option we have right know which choose between detailed and basic view with the same criterias)